application/values-test.yaml

```diff
@@ -386,7 +386,7 @@ rbac:
   # Service Account to use by pods
   serviceAccount:
     enabled: true
-    name: ""
+    name: "application"
```

Comment:
this shouldn't be hardcoded instead generated dynamically

Reply:
These values are used to ensure the templates are rendered properly during a `helm install`, right? What's your idea for generating this value dynamically? :thinking: